scripts/ytccf.sh

```diff
@@ -35,14 +36,75 @@ key_bindings="
       alt-u: mark last watched video as unwatched
       alt-h: show help"
 
-check_cmd() {
+declare -r -x THUMBNAIL_DIR=$HOME/.cache/ytccf/thumbnails
+declare -r -x UEBERZUG_FIFO="$(mktemp --dry-run --suffix "fzf-$$-ueberzug")"
```

Comment:
I don't really care, but do you really want a German name for this variable?

Reply:
I guess so, because it's the name of the tool to display the thumbnails :wink: 
https://github.com/seebye/ueberzug

Reply:
:facepalm: Sorry, I should have suspected something like this.